Makefile

```diff
@@ -0,0 +1,36 @@
+sleep:
+	sleep 10
+
+run: start sleep setup
+
+start:
+	echo -e "\033[0;32m Starting Inspire. \033[0m"
+	docker-compose up -d
+	echo -e "\033[0;32m Inspire Started. \033[0m"
+
+stop:
+	docker-compose down
+	echo -e "\033[0;32m Inspire Stopped. \033[0m"
+
+ui:
+	docker-compose up -d ui
+	sleep 5
+
+setup: hep-setup sleep load-records
+
+hep-setup:
+	echo -e "\033[0;32m Starting setup Inspire. \033[0m"
+	docker-compose -f docker-compose.yml exec hep-web ./scripts/setup
+	echo -e "\033[0;32m Finished setup Inspire. \033[0m"
+
+load-records:
+	echo -e "\033[0;32m Loading records. \033[0m"
+	docker-compose exec hep-web inspirehep importer demo-records
+
+next-setup:
+	docker-compose -f docker-compose.yml exec next-web inspirehep db create
```

Comment:
maybe we can exec all those setup steps actually (these are from `setup_inspire_next` script)? Cause for example we don't create HP indices with hep commands etc.
```
poetry run inspirehep db drop --yes-i-know
poetry run inspirehep alembic upgrade
poetry run inspirehep index destroy --force --yes-i-know
poetry run inspirehep index init
poetry run inspirehep fixtures init
```